swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/model/Account.kt

```diff
@@ -43,7 +44,8 @@ data class SpinnakerAccount(
   override val type: String,
   override val name: String,
   override val edda: String?,
-  override val regions: List<Region>?
+  override val regions: List<Region>?,
+  override val environment: String?
```

Comment:
When would environment be null? When this is not Edda backed, and instead a generic account?

Reply:
It is my understanding that account environment is primarily a netflix construct that isn't used universally in OSS installs, but not certain. I'm going to remove nullability here for now, in response to Rob's feedback.